One of the tests is failing:
```
In file included from /workdir/modules/lib/tensorflow/./tensorflow/lite/micro/micro_mutable_op_resolver.h:24,
--
  | from /workdir/modules/lib/tensorflow/./tensorflow/lite/micro/all_ops_resolver.h:16,
  | from /workdir/zephyr/samples/modules/tensorflow/hello_world/src/main_functions.cc:19:
  | /workdir/modules/lib/tensorflow/./tensorflow/lite/kernels/internal/compatibility.h:81:19: error: 'int8_t' in namespace 'std' does not name a type
  | 81 \| using int8 = std::int8_t;
  | \|                   ^~~~~~
```

I guess we must not add the headers to the search path if libstdc++ is around (so add only if CONFIG_LIB_CPLUSPLUS != y?)

--- a/subsys/cpp/CMakeLists.txt
+++ b/subsys/cpp/CMakeLists.txt
@@ -6,6 +6,12 @@ zephyr_sources(
   cpp_dtors.c
 )
 
+if (NOT CONFIG_LIB_CPLUSPLUS)
+zephyr_system_include_directories(
+	include
+)
+endif()
+
 if (NOT CONFIG_LIB_CPLUSPLUS AND
         (NOT CONFIG_MINIMAL_LIBC OR
         (CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE GREATER 0)))

--- a/subsys/cpp/include/cstddef
+++ b/subsys/cpp/include/cstddef
@@ -1,0 +1,11 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * @brief Stub header allowing compilation of `#include <cstddef>`
+ */

--- a/subsys/cpp/include/cstdint
+++ b/subsys/cpp/include/cstdint
@@ -1,0 +1,56 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * @brief Stub header allowing compilation of `#include <cstdint>`
+ */
+
+#ifndef ZEPHYR_SUBSYS_CPP_INCLUDE_CSTDINT_
+#define ZEPHYR_SUBSYS_CPP_INCLUDE_CSTDINT_
+
+#include <stdint.h>
+
+namespace std {
+	using ::int8_t;
+	using ::int16_t;
+	using ::int32_t;
+	using ::int64_t;
+	using ::intmax_t;
+
+	using ::int_fast8_t;
+	using ::int_fast16_t;
+	using ::int_fast32_t;
+	using ::int_fast64_t;
+
+	using ::int_least8_t;
+	using ::int_least16_t;
+	using ::int_least32_t;
+	using ::int_least64_t;
+
+	using ::uint8_t;
+	using ::uint16_t;
+	using ::uint32_t;
+	using ::uint64_t;
+	using ::uintmax_t;
+
+	using ::uint_fast8_t;
+	using ::uint_fast16_t;
+	using ::uint_fast32_t;
+	using ::uint_fast64_t;
+
+	using ::uint_least8_t;
+	using ::uint_least16_t;
+	using ::uint_least32_t;
+	using ::uint_least64_t;
+
+	using ::intptr_t;
+	using ::uintptr_t;
+}
+
+#endif /* ZEPHYR_SUBSYS_CPP_INCLUDE_CSTDINT_ */
+

